cmd/juju/caas/add_test.go

```diff
@@ -389,12 +389,12 @@ func (s *addCAASSuite) TestRegionFlag(c *gc.C) {
 		{
 			title:          "missing cloud",
 			regionStr:      "/region",
-			expectedErrStr: `validating cloud region "/region": cloud region "/region" not valid`,
+			expectedErrStr: `parsing region option: host cloud region "/region" not valid`,
```

Comment:
we need a lot more test cases for the new args, whether region is required etc

Reply:
added more tests